docs/photutils/psf.rst

```diff
@@ -481,6 +481,159 @@ Consider the previous example after the line
     plt.colorbar(orientation='horizontal', fraction=0.046, pad=0.04)
     plt.show()
 
+Fitting additional parameters
+-----------------------------
+
+The PSF photometry classes can also be used to fit additional parameters
+simultaneously, for instance, the PSF width and the sky background.
+
+Consider the case in which one would like to fit the ``sigma`` parameter in
+``IntegratedGaussianPRF``. First, let us instantiate a psf model object:
+
+.. doctest-skip::
+
+    >>> gaussian_prf = IntegratedGaussianPRF()
+
+The attribute ``fixed`` for the ``sigma`` parameter is set to ``True`` by
+default, i.e., ``sigma`` is not considered during the fitting process.
+Let's first change this behavior:
+
+.. doctest-skip::
+
+    >>> gaussian_prf.sigma.fixed = False
+
+In addition, we need to indicate the inital guess which will be used in during
+the fitting process. By the default, the initial guess is taken as the default
+value of ``sigma``, but we can change that by doing:
+
+.. doctest-skip::
+
+    >>> gaussian_prf.sigma.value = 2.05
+
+Now, let's create an artificial image which has a brighter star and one
+overlapping companion at a distance close enough so that the detection
+algorithms won't be able to identify it, and hence we should use
+``IterativelySubtractedPSFPhotometry`` to reduce the fainter star as well.
+Also, note that both of the stars have ``sigma=2.0``.
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    from photutils.datasets import make_random_gaussians, make_noise_image
+    from photutils.datasets import make_gaussian_sources
```

Comment:
Note that the names for the `datasets` functions might be changing in #474 , so there may need to be some fixing depending on the order in which these PR's get merged.